README.md

```diff
@@ -4,6 +4,26 @@ This action uses aws cli to login to EKS and deploy a helm chart.
 
 ## Inputs
 
+### `atomic`
+
+If set, upgrade process rolls back changes made in case of failed upgrade.
```

Comment:
```suggestion
If set, upgrade process rolls back changes made in case of failed upgrade. Must be either `'true'` or `'false'`, defaults to `'true'`.
```